pyproject.toml

```diff
@@ -52,7 +52,7 @@ Werkzeug = "~3.0.6"
 setuptools = "^75.2.0"
 honcho = "^2.0.0"
 detect-secrets = "^1.5.0"
-moto = {extras = ["iam"], version = "^5.0.21"}
+moto = {version = "^5.0.22", extras = ["s3,iam"]}
```

Comment:
Question: Should `extras` not be a list, rather than a comma-separated string?

Reply:
I dunno - this was set by poetry add and seems to work

Reply:
Fixed in [e1ecdb3](https://github.com/ONSdigital/dis-wagtail/pull/46/commits/e1ecdb3ebf071725c4e1e98515bbdf27d20fadca)